src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -32,6 +32,23 @@ def destroy
     render(partial: 'webui/comment/comment_list', locals: { commentable: @commentable }, status: status)
   end
 
+  def update
+    comment = Comment.find(params[:id])
+    authorize comment, :update?
+
+    comment.assign_attributes(permitted_params)
+    @commentable = comment.commentable
+
+    status = if comment.save
+               flash.now[:success] = 'Comment updated successfully.'
+               :ok
+             else
+               flash.now[:error] = "Failed to update comment: #{comment.errors.full_messages.to_sentence}."
+               :unprocessable_entity
+             end
+    render(partial: 'webui/comment/comment_list', locals: { commentable: @commentable }, status: status)
+  end
```

Comment:
I think you can use a [respond_to](https://api.rubyonrails.org/classes/ActionController/MimeResponds.html#method-i-respond_to) block to limit responses to `xhr` requests, as we just use it for the 'Edit' comment button and anywhere else.

Reply:
Hm, you are right, on it :eyes: